### DISCUSSION TITLE
Supabase branching and Realtime

### DISCUSSION THREAD

**Discussion by jaisonj708** [supabase/supabase#28718] [General]
My realtime schema is empty in a preview branch, which I think is preventing me from using the realtime service.  Does anyone know the steps to enabling realtime on a Preview branch?

**Comment by Garett-MacGowan** [comment]
@jaisonj708 I'm facing the same issue. Did you end up resolving this?

**Reply by sweatybridge** [reply]
We have released a fix for this last week. Could you double check if this is still happening?
